schema.sql

```diff
@@ -8,4 +8,18 @@ CREATE TABLE animals (
 );
 
 -- Vet clinic database: query and update animals table
-ALTER TABLE animals ADD COLUMN species VARCHAR;
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR;
+
+-- Vet clinic database: query multiple tables
+
+ CREATE TABLE owners(id INT GENERATED ALWAYS AS IDENTITY, full_name VARCHAR, age INT, PRIMARY KEY(id));
+ CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR, PRIMARY KEY(id));

```

Comment:
- [ ] Just as i suggested in the `owners` table, kindly set a limit to the `name` variable 👍